components/motor/gpio/basic.go

```diff
@@ -305,5 +308,5 @@ func (m *Motor) ResetZeroPosition(ctx context.Context, offset float64, extra map
 
 // GoTillStop is not supported.
 func (m *Motor) GoTillStop(ctx context.Context, rpm float64, stopFunc func(ctx context.Context) bool) error {
-	return motor.NewGoTillStopUnsupportedError("(name unavailable)")
+	return motor.NewGoTillStopUnsupportedError(m.motorName)
```

Comment:
Please add motor name as an input to the other common motor errors the components/motor/errors.go file